lib/containers/podman.pm

```diff
@@ -38,4 +40,46 @@ sub get_storage_driver {
     return $storage;
 }
 
+sub switch_network_backend {
+    my ($self, $net) = @_;
```

Comment:
Let's use $backend instead of $net as the latter may be anything from network, etc.

Reply:
in such a case I would rename not only variable but function too